library/billing/src/Claim.php

```diff
@@ -1,59 +1,63 @@
 <?php
-/** @package OpenEMR
- *  @link http://www.open-emr.org
- *  @author Rod Roark <rod@sunsetsystems.com>
- *  @copyright Copyright (c) 2009 Rod Roark <rod@sunsetsystems.com>
- *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+/* Claim Class
+ *
+ * @package OpenEMR
+ * @author Rod Roark <rod@sunsetsystems.com>
+ * @author Stephen Waite <stephen.waite@cmsvt.com>
+ * @copyright Copyright (c) 2009 Rod Roark <rod@sunsetsystems.com>
+ * @copyright Copyright (c) 2017 Stephen Waite <stephen.waite@cmsvt.com>
+ * @link https://github.com/openemr/openemr/tree/master
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
- 
-require_once(dirname(__FILE__) . "/invoice_summary.inc.php");
 
-// This enforces the X12 Basic Character Set. Page A2.
-//
-function x12clean($str)
+namespace OpenEMR\Billing;
```

Comment:
below this namespace line (with a blank line above and below it), add the following:
``````
use InsuranceCompany;
``````